javascripts/next.meeting.js

```diff
@@ -0,0 +1,57 @@
+/* 
+* Dispay Next Meeting
+* i.e. calculate nearest first or third tuesday 
```

Comment:
Ooohhh. I see. This may need revisiting. We are looking into some alternate schedules and may deviate from the 1st & 3rd. If you can hook into MeetUp data, that would be ideal. Alternately, maybe a data file that we use to schedule explicitly?